CRM/Core/BAO/Domain.php

```diff
@@ -142,7 +142,7 @@ public static function edit(&$params, &$id) {
    */
   public static function create($params) {
     $domain = new CRM_Core_DAO_Domain();
-    $domain->copyValues($params);
+    $domain->copyValues($params, TRUE);
```

Comment:
Remind me what the second parameter does here?

Reply:
it's new - we added array handling to that function - providing the metadata is defined - new in the xml